### DISCUSSION TITLE
GitHub is Down, switching to GitLab

### DISCUSSION THREAD

**Discussion by pablodiazt** [community/community#140442] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I'm incredibly frustrated and upset, switching to GitLab from GitHub.

I been dealing with tons of crap from GitHub in the past few days, which has prevented me from working. Furthermore, I'm unable to ssh or https access GH from command line terminal on my macbook pro. I can't access GH from any browser except from Safari..

This is very upsetting and preventing my business from moving forward.

Haven't been able to solve this in days. Is GitHub going to reimburse and even pay for this damages?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @pablodiazt ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by milad-2003** [comment]
Hi there!
What is the exact problem and error that you get?
Can you open "Git Log" and write down the errors?

**Comment by pablodiazt** [comment, marked as answer]
Thanks so much for your reply and letting me know. Apologies about that. I
was able to resolve the issue after much work.

Issue had to do with keys as they can complicate things.


Thanks again for your help!

Pablo
Founder & CEO @ *Blossend* <https://blossend.com>
